report/_toc.yml

```
@@ -13,13 +13,13 @@ parts:
       - file: verification/methods
       - file: verification/cases
         sections:
-        - file: verification/depleting_source
+        - file: verification/mes/depleting_source
         - file: verification/mms/simple
         - file: verification/mms/fluxes
         - file: verification/mms/discontinuity
         - file: verification/mms/trapping
         - file: verification/mms/heat_transfer
-        - file: verification/mes_radioactive_decay
+        - file: verification/mes/mes_radioactive_decay
```

```suggestion
        - file: verification/mes/radioactive_decay
```

mes is redundant here